core/Symbols.h

```diff
@@ -654,7 +654,7 @@ class ClassOrModule final {
 
     void addMixinAt(ClassOrModuleRef sym, std::optional<uint16_t> index);
 };
-CheckSize(ClassOrModule, 128, 8);
+CheckSize(ClassOrModule, 120, 8);
```

Comment:
Whoa, what is this from?  Did `UnorderedMap` get smaller?

Reply:
The commit message that introduced this shows some of my debugging notes, but I was never able to pinpoint an exact abseil commit that blames to this change.

Reply:
https://github.com/sorbet/sorbet/pull/7884/commits/083d8114bafb0952b77da8b21806857cb9b3884b